griptape/events/event_listener.py

```diff
@@ -12,6 +13,10 @@
 
 T = TypeVar("T", bound=BaseEvent)
 
+_active_listeners: contextvars.ContextVar[list[EventListener]] = contextvars.ContextVar(
+    "active_event_listeners", default=[]
+)
+
```

Comment:
Just curious, why did you choose to put `_active_listeners` in in `event_listener.py` as opposed to `event_bus.py`? I would imagine if you did it the other way this file would be almost unchanged.

I don't have a strong opinion, just want to hear the reasoning